controls/src/main/res/drawable/seeker.xml

```diff
@@ -29,5 +29,8 @@
     <item
         android:id="@android:id/progress"
         android:drawable="@drawable/ic_thumbseek"/>
+    <item
+        android:id="@+id/ad_marks"
+        android:drawable="@drawable/ic_thumbseek_background"/>
```

Comment:
Why do we need 2 `ic_thumbseek_background` rework this, one. If there will be no advertisement. We will have bolder seekbar.